src/Build/project.json

```diff
@@ -3,6 +3,8 @@
     "net46": {
       "dependencies": {
         "Microsoft.VisualStudio.Setup.Configuration.Interop": "1.14.114",
+        "NuGet.Commands": "4.5.0",
+        "NuGet.Protocol": "4.5.0",
```

Comment:
Do we need to get the NuGet team to add us to their automatically-drop-new-NuGet-assemblies PRs?

Reply:
I'm not sure what this is?

Reply:
They have a bot that creates PRs like https://github.com/dotnet/sdk/pull/1851. I was wondering if we needed to get them to target us too. Hopefully not, since we don't need the very latest . . . until we adopt their version of the resolverEx thing.